geometric_kernels/frontends/pytorch/gpytorch.py

```diff
@@ -1,63 +1,96 @@
 """
-Pytorch kernel wrapper
+GPyTorch kernel wrapper
 """
 import gpytorch
-import numpy as np
 import torch
 
 from geometric_kernels.kernels import BaseGeometricKernel
 from geometric_kernels.spaces.base import Space
 
 
-class GPytorchGeometricKernel(gpytorch.kernels.Kernel):
+class GPyTorchGeometricKernel(gpytorch.kernels.Kernel):
     """
-    Pytorch wrapper for `BaseGeometricKernel`
+    GPyTorch wrapper for `BaseGeometricKernel`
+
+    Note: you cannot change trainable_nu after constructing the object.
```

Comment:
It's not clear in this context, what `trainable_nu` is. I'd suggest writing a docstring and explaining it there

Reply:
Done.